.github/workflows/deployment-template.yml

```diff
@@ -37,8 +45,8 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Step 1 - Replace Tokens
-        if: ${{inputs.deploy-env}} == 'preview'
+      - name: Step 1 - Replace Tokens for helm chart configuration
+        if: ${{inputs.deploy-env == 'preview' }}
```

Comment:
nit: add one space on the left as well `if: ${{ inputs.deploy-env == 'preview' }}`